docs/blocks-storage/production-tips.md

```diff
@@ -17,6 +17,16 @@ If your Cortex cluster has many tenants or ingester is running with a long `-blo
 
 The rule of thumb is that a production system shouldn't have the `file-max` ulimit below `65536`, but higher values are recommended (eg. `1048576`).
 
+### Ingester disk space
+
+Ingesters create blocks on disk as samples come in, then every 2 hours (configurable) they cut off those blocks and start a new block.
+
+We typically configure ingesters to retain these blocks for longer, to allow time to recover if something goes wrong uploading to the long-term store and to reduce work in queriers - more detail [here](#how-to-estimate--querierquery-store-after).
+
+If you configure ingesters with `-blocks-storage.tsdb.retention-period=24h`, a rule of thumb for disk space required is to take the number of timeseries after replication and multiply by 30KB.
```

Comment:
I estimate it some time out for an internal TCO calculator and the number I've got was:
- 500 bytes / series / hour for compacted blocks
- 12 bytes / sample for the WAL

Assuming we have 4h of data in the WAL and 24h data in blocks, and assuming a scrape interval period of 15s, we would have:
- Blocks: 500 bytes * 24 = 11.7KB
- WAL: 12 * (4 * 60 * 4) = 11KB

The roughtly estimated total is 23KB, which is same order of magnitude of your 30KB. So 30KB LGTM.